tests/frontend/test_search.py

```diff
@@ -152,6 +163,21 @@ def test_selected_result_is_highlighted(base_url, selenium, variables):
     assert page.search.highlighted_suggestion
 
 
+@pytest.mark.nondestructive
+def test_search_box_character_limit(base_url, selenium, variables):
+    page = Home(selenium, base_url).open()
+    term = ''
```

Comment:
you could use the `get_random_string()` method from our scripts/reusables file, to define a 100 char term.